tokio/src/runtime/blocking/mod.rs

```diff
@@ -35,5 +36,8 @@ cfg_not_blocking_impl! {
         pub(crate) fn spawner(&self) -> &BlockingPool {
             self
         }
+
+        pub(crate) fn shutdown(&mut self, _duration: Option<Duration>) {
+        }
```

Comment:
I'm guessing this formatting is what rustfmt is upset about.

Reply:
turns out it isn't!